composer/callbacks/speed_monitor.py

```diff
@@ -83,30 +86,47 @@
         'int8': 130e12,
         'int4': 260e12,
     },
+    'trn1': {
+        'fp32': 190e12 / 2,
+        'tf32': 190e12 / 2,
+        'fp16': 190e12 / 2,
+        'amp_fp16': 190e12 / 2,
```

Comment:
Curious if you know why fp16 == fp32?

Reply:
Thanks for the catch, I forgot to commit a change, fp32 is actually 4 times slower than fp16.